frontend/src/pages/Dashboard/index.tsx

```diff
@@ -28,6 +28,14 @@ export default function DashboardPage() {
       colorClass: "card-blue",
       stats: "5 active sets",
     },
+    {
+      title: "Search",
+      description: "Find study sets and flashcards",
+      icon: "🔍",
+      path: "/search",
+      colorClass: "card-teal",
+      stats: "Explore",
+    },
```

Comment:
this should ideally just be a search bar on top of the dashboard, but this is fine.